### PR TITLE
requirePaddingNewLinesBeforeLineComments fix for file starting with line comment

### LINKED ISSUE #1261: requirePaddingNewLinesBeforeLineComments breaks if no previous token
```
if (prevToken.type === 'Line') {
    return;
}

if (allowFirstAfterCurly && prevToken.type === 'Punctuator' && prevToken.value === '{') {
    return;
}
```

If `prevToken` is undefined here, this code will break.  I happen to have some files where, likely, there's a newline to start the file (so `comment.loc.start.line` is not 1).  Chances are, there should be an early return here if `prevToken` is `undefined`.  

In any case, running jscs on a large project only to see:

`Warning: Cannot read property 'type' of undefined Use --force to continue.` isn't ideal.

<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/11061838-requirepaddingnewlinesbeforelinecomments-breaks-if-no-previous-token?utm_campaign=plugin&utm_content=tracker%2F281640&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F281640&utm_medium=issues&utm_source=github).
</bountysource-plugin>

### PR DESCRIPTION
**Pull request by TheSavior**
Fix for file starting with line comment.

Fixes #1261

### CODE REVIEW

File: lib/rules/require-padding-newlines-before-line-comments.js [hunk 1]
```diff
@@ -83,13 +83,9 @@ module.exports.prototype = {
         var allowFirstAfterCurly = this._allowFirstAfterCurly;
 
         file.iterateTokensByType('Line', function(comment) {
-            if (comment.loc.start.line === 1) {
```
Comment (zxqfox) [inline lib/rules/require-padding-newlines-before-line-comments.js]: Does it affects shebangs?
```
#!/usr/bin/env node
// etc.?
```

Comment (TheSavior) [inline lib/rules/require-padding-newlines-before-line-comments.js]: What do you feel like the expected behavior here? I would expect a blank line between the shebang and comment.

Comment (zxqfox) [inline lib/rules/require-padding-newlines-before-line-comments.js]: I'd say
```
#!/usr/bin/env

// line comment
// etc.
```
But
```
// line comment
// etc.
```

Well, it's very rare and hard to implement case (unless we have a isShebanged flag in jsfile). Nvm! ;-)

Comment (zxqfox) [inline lib/rules/require-padding-newlines-before-line-comments.js]: Am I drunk? Not sure. Looks like it works well already ;-)

**Pull request closed without merging**